micro_sam/sam_annotator/image_series_annotator.py

```diff
@@ -64,7 +71,7 @@ def image_series_annotator(
     """Run the 2d annotation tool for a series of images.
 
     Args:
-        input_files: List of the file paths for the images to be annotated.
+        image_files: List of the file paths or list of (set of) slices for the images to be annotated.
```

Comment:
Let's call this `images`. That's more correct now since we can have both files and arrays.

Reply:
Fixed. Thanks!